### ISSUE TITLE
Python wrapper

### ISSUE THREAD

**Issue by cocool97** [cocool97/adb_client#68]
Check if using [`pyo3`](https://pyo3.rs/) library could help us creating a native Python module in Rust.

If this is possible, this would be nice to also publish this crate on pypi.

**Comment by tan9-net** [comment]
Yes, you can use the pyo3 library to create a native Python module in Rust. pyo3 is a powerful library that allows you to write native Python extensions in Rust. This enables you to leverage Rust's performance and safety features while providing a Python interface for your code.

**Comment by cocool97** [comment]
Would you like to work on a POC to use this library in Python ?

This is not in my priorities for the next few weeks

**Comment by cocool97** [comment]
Work done and merged upstream
